File: src/kyu_6/make_a_window/python/make_a_window_test.py
import codewars_test as test
from solution import make_a_window

@test.describe('Example Tests')
def example_tests():
    @test.it('Example Test Case')
    def example_test_case():
        test.assert_equals(make_a_window(3), "---------\n|...|...|\n|...|...|\n|...|...|\n|---+---|\n|...|...|\n|...|...|\n|...|...|\n---------")

File: src/kyu_6/make_a_window/js/makeAWindowTest.js
const { assert } = require('chai');
const makeAWindow = require("./makeAWindow");

describe('Make window tests', () => {
    it('Sample test', () => {
        assert.strictEqual(makeAWindow(3), "---------\n|...|...|\n|...|...|\n|...|...|\n|---+---|\n|...|...|\n|...|...|\n|...|...|\n---------");
    });
});